mkdocs.yml

```diff
@@ -2,7 +2,7 @@ site_name: fastlane docs
 theme: readthedocs
 site_author: "fastlane team"
 site_description: "Documentation for fastlane tools, the easiest way to automate building and releasing your iOS and Android apps"
-strict: true
+# strict: true
```

Comment:
Yeah, for now, since deploy will make sure there are no deadlinks, and we currently do have dead links.